main.js

```diff
@@ -141,23 +161,144 @@ function createJitsiMeetWindow() {
     windowState.manage(mainWindow);
     mainWindow.loadURL(indexURL);
 
+    const isWindowAlwaysOnTop = getWindowAlwaysOnTopFromStore();
+
+    //
+    // This is required for some features
+    // Google Sign In - Start Youtube Live Stream, Settings -> Calendar
+    // Microsoft Sign In - Settings -> Calendar
+    // Dropbox Sign In - Start Recording Feature.
+    //
     initPopupsConfigurationMain(mainWindow);
-    setupAlwaysOnTopMain(mainWindow);
+
+    if (isWindowAlwaysOnTop) {
+        setupAlwaysOnTopMain(mainWindow);
```

Comment:
Hey @gorance-teletrader 
Hey @gorance-teletrader 

I just took another look and I found that `setupAlwaysOnTopRender` is still being executed in Conference component. Can you make that prop dependent? 😄 

Also, please check what is the effect of keeping `setupAlwaysOnTopMain` and just not doing `setupAlwaysOnTopRender` if the prop it false. 

It might clean up main a lot. 😄